package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
-    "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "seed": "PGDATABASE=nc_news node ./db/seeds/run-seed.js",
+    "test": "PGDATABASE=nc_news jest",
```

Comment:
From this change in your package.json I'm slightly concerned that you don't have your connection.js file setup to use `ENV`, the scripts should be fine as they were and the connection.js file will use the `dotenv` to connect to the correct database